src/Routers/FilesRouter.js

```diff
@@ -166,16 +166,26 @@ export class FilesRouter {
         // update fileSize
         const bufferData = Buffer.from(fileObject.file._data, 'base64');
         fileObject.fileSize = Buffer.byteLength(bufferData);
+        // prepare file options
+        const fileTags = fileObject.file._tags;
```

Comment:
Maybe more concise:
```
const fileTags = Object.keys(fileObject.file._tags).length > 0 ? fileObject.file._tags : {};
Object.assign(fileOptions, fileTags);
```

Reply:
I applied this. But Codecov is still whining because of some files that I did not touch. Do you have any idea about this?

Reply:
See the impacted files section above in the Codecov message. The coverage increased in the `FilesRouter.js` file. But there are other files.